.github/workflows/lint.yml

```
@@ -0,0 +1,40 @@
+name: lint
+
+on:
+  push:
+    tags:
+      - v*
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  golangci:
+    name: lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@v2
+        with:
+          version: v1.29
+
+          # Optional: golangci-lint command line arguments.
+          # args: --issues-exit-code=0
+
+          # Optional: show only new issues if it's a pull request. The default value is `false`.
+          # only-new-issues: true
+
+          # Optional: if set to true then the action will use pre-installed Go
+          # skip-go-installation: true
+  checklicenses:
+    name: checklicenses
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: check license headers
+        run: |
+          set -exu
+          go get -u github.com/u-root/u-root/tools/checklicenses
+          go install github.com/u-root/u-root/tools/checklicenses
+          go run github.com/u-root/u-root/tools/checklicenses -c .ci/checklicenses_config.json
```

I don't think we need all 3 of these, `go get` already does the install, this should be enough right ?
```suggestion
          go get -u github.com/u-root/u-root/tools/checklicenses
          $(go env GOPATH)/bin/checklicenses -c .ci/checklicenses_config.json
```

---

correct, fixing it

---

done